sdk/rust/src/consensus/zmq_driver.rs

```diff
@@ -197,6 +197,7 @@ pub fn register(
                         break;
                     }
                     ConsensusRegisterResponse_Status::NOT_READY => {
+                        ::std::thread::sleep(::std::time::Duration::from_millis(1500));
```

Comment:
This would be better done as an exponential backoff up to a limit. Why 1500?

Reply:
The current value was due to testing, while trying to find the deadlocks with the validator changes.  I agree an exponential backoff would make more sense here.